policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/work_premium/general_work_premium/couple/amount/with_children.yaml

```
@@ -0,0 +1,12 @@
+description: Quebec limits its work premium credit to eligible couple with at least one child at or below this amount.
+values:
+  2022-01-01: 3_449.5
+metadata:
+  unit: currency-CAD
+  label: Quebec maximum anual work premium amount for couple with at least one child
```

Add `period: year` instead (here and elsewhere)
```suggestion
  label: Quebec maximum work premium amount for couple with at least one child
```